jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildDockerTask.java

```diff
@@ -88,9 +88,12 @@ public void buildDocker()
       throws InvalidImageReferenceException, IOException, BuildStepsExecutionException,
           CacheDirectoryCreationException, MainClassInferenceException,
           InferredAuthRetrievalException {
-    DockerClient dockerClient =
-        DockerClient.newClient(dockerClientParameters.getExecutable(), null);
-    if (!dockerClient.isDockerInstalled()) {
+    Path dockerExecutable = dockerClientParameters.getExecutable();
+    DockerClient.Builder dockerClientBuilder = new DockerClient.Builder();
```

Comment:
I lean towards having a static factory to create a builder (`DockerClient.builder()`) over using the inner class constructor.